src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int LENGTH_MULTIPLIER = 2;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K, V>[] table;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
 
+        resize();
+        int index = hashCode(key);
```

Comment:
As Leonid mentioned before, method name of hashCode() doesn't represent what method is doing.
Should be something like calculateBucketIndex, getIndexFromHashCode, anything else that giving you clue what is going on without need to look into method itself